PlaceRouteHierFlow/router/A_star.cpp

```diff
@@ -131,6 +134,13 @@ int A_star::Manhattan_distan(int sindex, Grid &grid) {
   int dis = *it;
 
   return dis;
+  **/
+  int max_dis = 0;
+  for (int i = 0; i < (int)dest.size(); i++) {
+    int temp_dis = abs(grid.vertices_total[sindex].x - grid.vertices_total[dest[i]].x) + abs(grid.vertices_total[sindex].y - grid.vertices_total[dest[i]].y);
```

Comment:
This code change is easy to understand. 
Old code: store all distance into a set, pick the last one. Most time usage is "insert" function
New code: update max distance into an int, without set.